spec/abstract/SchedContext_A.thy

```diff
@@ -309,34 +309,40 @@ definition
   sched_context_update_consumed :: "obj_ref \<Rightarrow> (time,'z::state_ext) s_monad" where
   "sched_context_update_consumed sc_ptr \<equiv> do
     sc \<leftarrow> get_sched_context sc_ptr;
-    set_sc_obj_ref sc_consumed_update sc_ptr 0;
-    return (sc_consumed sc)
+    consumed <- return (sc_consumed sc);
```

Comment:
I updated the spec for `sched_context_update_consumed` in my "smaller ipc sorries" PR. I'm definitely happy for you to go first if you'd like, but there is one change that Curtis has made to the C which needs to be included. We need to add `ticks_to_us` before `max_ticks_to_us` in the "then" branch. The C change is part of Curtis's PR #1813 on the SEL4 Bitbucket

Reply:
Sounds good :)